packages/extension-chakra-storefront/src/components/tooltip/index.jsx

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
For ease of use, created a closed component composition for the Tooltip component as suggest in the Chakra doc